src/BuiltInTools/DotNetDeltaApplier/StartupHook.cs

```diff
@@ -59,7 +59,7 @@ public static void Initialize()
                 agent.Reporter.Report("Writing capabilities: " + agent.Capabilities, AgentMessageSeverity.Verbose);
```

Comment:
This is getting to be a pretty long delegate.  Would it make sense to extract it into a method that takes (it appears) just a CancellationToken?